Sources/LegacyValet/Public/VALLegacySecureEnclaveValet.h

```diff
@@ -14,7 +14,7 @@
 //  limitations under the License.
 //
 
-#import "VALLegacyValet.h"
```

Comment:
whoops. Xcode is warning about these malformed imports now, which is long overdue!